server/spec/models/host_node_stat_spec.rb

```diff
@@ -11,6 +11,21 @@
   it { should have_index_for(host_node_id: 1).with_options(background: true) }
   it { should have_index_for(host_node_id:1, created_at: 1).with_options(background: true) }
 
+  describe '.latest' do
+    let(:grid) { Grid.create!(name: 'test-grid') }
+    let(:node) { HostNode.create!(grid: grid, name: 'test-node') }
+
+    it 'returns latest stat item' do
+      node.host_node_stats.create
+      last = node.host_node_stats.create
+      expect(described_class.latest).to eq(last)
```

Comment:
Confirm that the spec works, aka even with just the two items, removing the `asc(:id)` allows this to fail:

```
  1) HostNodeStat.latest returns latest stat item
     Failure/Error: expect(described_class.latest).to eq(last)
       
       expected: #<HostNodeStat _id: 5955f959de3578646bd0b488, created_at: 2017-06-30 07:10:17 UTC, load: nil, memory: nil, filesystem: nil, usage: nil, cpu: nil, network: nil, grid_id: nil, host_node_id: BSON::ObjectId('5955f959de3578646bd0b484')>
            got: #<HostNodeStat _id: 5955f959de3578646bd0b487, created_at: 2017-06-30 07:10:17 UTC, load: nil, memory: nil, filesystem: nil, usage: nil, cpu: nil, network: nil, grid_id: nil, host_node_id: BSON::ObjectId('5955f959de3578646bd0b484')>
       
       (compared using ==)
       
       Diff:
       @@ -1,2 +1,2 @@
       -#<HostNodeStat _id: 5955f959de3578646bd0b488, created_at: 2017-06-30 07:10:17 UTC, load: nil, memory: nil, filesystem: nil, usage: nil, cpu: nil, network: nil, grid_id: nil, host_node_id: BSON::ObjectId('5955f959de3578646bd0b484')>
       +#<HostNodeStat _id: 5955f959de3578646bd0b487, created_at: 2017-06-30 07:10:17 UTC, load: nil, memory: nil, filesystem: nil, usage: nil, cpu: nil, network: nil, grid_id: nil, host_node_id: BSON::ObjectId('5955f959de3578646bd0b484')>
       
     # ./spec/models/host_node_stat_spec.rb:21:in `block (3 levels) in <top (required)>'
```